webui/react/src/pages/F_ExpList/glide-table/contextMenu.tsx

```diff
@@ -66,9 +70,10 @@ export const TableContextMenu: React.FC<TableContextMenuProps> = ({
       }}>
       <ExperimentActionDropdown
         experiment={experiment}
+        handleUpdateExperimentList={handleUpdateExperimentList}
         makeOpen={open}
         onComplete={onComplete}
-        onVisibleChange={onComplete}>
+        onVisibleChange={() => handleClose()}>
```

Comment:
nit: why is this an arrow function instead of just passing `handleClose`?

Reply:
This is because `handleClose` takes an `event` argument while `onVisibleChange` takes a `boolean` argument